modules/andes-core/broker/src/main/java/org/wso2/andes/subscription/SubscriptionStore.java

```diff
@@ -184,6 +209,44 @@ public SubscriptionStore() throws AndesException {
     }
 
     /**
+     * @param destination queue name
+     * @param isTopic     FALSE if checking queues
```

Comment:
FALSE if checking queues doesn't make sense to me. What about a comment like whether a topic or a queue? WDYT?